### PR TITLE
Update the helm-component example to demonstrate the automated rendering feature

### PR DESCRIPTION
**Pull request by nan-yu**

### CODE REVIEW

**Comment by nan-yu** [comment]
@janetkuo @haiyanmeng @Liujingfang1 
Please review the example when you get a chance.

File: helm-component/README.md [hunk 1]
```diff
@@ -1,151 +1,9 @@
...
-  ```console
-  Kubectl get namespace cert-namespace
-  ```
+This example provides two different ways of rendering and syncing your configurations
+- manual rendering:
+  this option requires you to render the configurations using the Helm CLI manually,
+  and check in the rendered configurations to your Git repository.
+  Config Sync will sync from the rendered output directly.
+- automated rendering:
+  Config Sync support automated rendering in versions 1.9.0 or later.
```
Comment (janetkuo) [inline helm-component/README.md:7]:
```suggestion
  Config Sync supports rendering Kustomize configurations and Helm Charts in versions 1.9.0 or later.
```

File: helm-component/README.md [hunk 2]
```diff
@@ -1,151 +1,9 @@
...
-- Delete the cert-manager namespace.
-  ```console
-  git rm manifests/namespace-cert-manager.yaml && git commit -m "remove the cert-manager namespace" && git push origin init
-  ````
-- Verify the namespace does not exist.
-  ```console
-  Kubectl get namespace cert-namespace
-  ```
+This example provides two different ways of rendering and syncing your configurations
+- manual rendering:
```
Comment (janetkuo) [inline helm-component/README.md:2]: Link to the manual rendering folder or README

File: helm-component/README.md [hunk 3]
```diff
@@ -1,151 +1,9 @@
...
+  this option requires you to render the configurations using the Helm CLI manually,
+  and check in the rendered configurations to your Git repository.
+  Config Sync will sync from the rendered output directly.
+- automated rendering:
```
Comment (janetkuo) [inline helm-component/README.md:6]: Link to the automated rendering folder or README

**Review by janetkuo — APPROVED** [review]
Just a few nits; LGTM otherwise

File: helm-component/README.md (hunk 1 above)
Comment (nan-yu) [reply helm-component/README.md]: DONE

File: helm-component/README.md (hunk 2 above)
Comment (nan-yu) [reply helm-component/README.md]: DONE

File: helm-component/README.md (hunk 3 above)
Comment (nan-yu) [reply helm-component/README.md]: DONE

**Pull request merged**